biome.json

```diff
@@ -13,6 +13,14 @@
     "enabled": true,
     "rules": {
       "recommended": false,
+      "a11y": {
+        "noBlankTarget": "error"
+      },
+      "correctness": {
+        "noUnreachable": "error",
+        "useHookAtTopLevel": "error",
```

Comment:
is this the only part of rules of hooks?

Reply:
react-hooks/rules-of-hooks does more than just putting things at the top level

Reply:
oh i see `react-hooks/exhaustive-deps` is separate

Reply:
I didn't include exhaustive deps because there are 5000 issues on the sentry repository. 🫡